OFEP-ofo-flagd-service.md

```diff
@@ -0,0 +1,33 @@
+## flagd service deployment driven by OpenFeature Operator
+
+## State: DRAFTING
+
+Currently, OpenFeature Operator (OFO) manages the deployment of flagd by appending it to a pod's containers (sidecar pattern), thereby allowing containers within the pod to route to it. This OFEP campaigns for the extension of OFO to manage the deployment of externally accessible flagd (e.g. from client side applications, or even server side applications that don't want to use the sidecar pattern).
+
+## Background
+
+The driving force behind this is to simplify the deployment of flagd for use by client side applications. An [OFEP was drafted](./OFEP-ofo-flagd-client-support.md) to achieve this but subsequently withdrawn (reasons noted within). This OFEP recognises the limitations of the previous, presenting a modular solution with a broader scope.
+
+## Proposal
+
+Introduce a FlagdService custom resource definition (CRD) and controller.
```

Comment:
I wonder if we could make this a bit more generic. Most flag vendors offer a remote evaluation service of some kind. We could start by only supporting flagd but it may be nice to have the option of adding additional tools if there's community interest.